packages/extension-ui/src/Popup/Derive/index.tsx

```diff
@@ -80,6 +79,7 @@ function Derive ({ match: { params: { address: parentAddress } } }: Props): Reac
         onChange={setAccount}
         parentAddress={parentAddress}
         parentPassword={parentPassword}
+        siblingCount={childrenCount(accounts, parentAddress)}
```

Comment:
That would be `siblingsCount` (plural).

Still I'm a bit unsure about the naming here. We have a bit of "business" logic here: the next derivation path should be equal to current count number of siblings.

The question is where should it go? Here, the DerivationPath, or perhaps a new file, that will include more complex rules for creating derivation paths in the future?

So perhaps here we should have `defaultDerivationPath={nextDerivationPath(childrenCount(accounts, parentAddress))}`